.travis.yml

```diff
@@ -0,0 +1,103 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+
+env:
+  global:
+    - JOBS=3
+    - MBEDTLS_VERSION=2.2.1
+    - MBEDTLS_PREFIX=~/opt/mbed
+    - MBEDTLS_LIBS="-L$MBEDTLS_PREFIX/lib -lmbedtls -lmbedx509 -lmbedcrypto"
+    - MBEDTLS_CFLAGS="-I$MBEDTLS_PREFIX/include"
+    - OPENSSL_VERSION=1.0.1t
+    - OPENSSL_PREFIX=~/opt/ssl
```

Comment:
currently I build both mbedtls and openssl in every build (it is cached by ccache).
if we will split single .travis.yml into several scripts in .travis folder, I think we can use conditional compilation (like building mbedtls only in appropriate builds)

putting those "if"s into .travis.yml pollute it, it is not readable after that